ewkcoffea/modules/selection_wwz.py

```diff
@@ -227,6 +260,7 @@ def add4lmask_wwz(events, year, isData, sample_name):
 
     # Filters
     filter_flags = events.Flag
+    # TODO: Should we have a different list of filters for Run3? Filter effect seems to be quite small
```

Comment:
I would say we should go with the ones Guillelmo applies. Do you have that information?

Reply:
I have them. there are some differences

Reply:
Please pass the is2022 flag for this.